node/lib/DeviceAccessor.ts

```diff
@@ -1,18 +1,89 @@
 
 import { DeviceInterface } from "./DeviceInterface";
-import { IDevice } from "./ITranslator";
+import { ITranslator } from "./ITranslator";
 
 import { EventEmitter } from "events";
 
 /**
  * Provides reflection-style access to device properties and methods via device interfaces.
  */
 export class DeviceAccessor {
```

Comment:
Onboarding related interface and its stubs are not yet defined. Is it because the onboarding design is in-progress?